napari/components/_interaction_box_mouse_bindings.py

```diff
@@ -50,6 +51,8 @@ def __init__(self, viewer, interaction_box_visual):
         self._fixed_vertex: int = None
         self._fixed_aspect: float = None
         self._layer_listening_for_affine = None
+        self._ini_transf = None
+        self._ini_transf_inv = None
```

Comment:
minor nitpick: `ini_transf` -> `initial_transform`

is this just the initial transform or is it more generally the layer transform? if so, you could call it `layer_transform`

because the inverse is available at `self._ini_transf.inverse` I'm not sure we need to store the inverse transform directly here

Reply:
First of all, thanks for the quick review :)
I have changed this to initial_transform

It is rather a part of the layer transform without the affine part, an initial transform of scaling, translation, so on. Therefore the name.

I store and build the inverse transform one time because otherwise the inverse is computed every time when the affine transform changes. I thought this should be not the case and can save computation time in this way.
Of course, the difference will not be huge. 

https://github.com/napari/napari/blob/906f772c01d6a4cb33d5d957f3cfd452b068145f/napari/utils/transforms/transforms.py#L504-L506

Reply:
this makes sense! agree that it's better not to recompute if it can be avoided